site/docs/components/file-drop-zone/index.mdx

```diff
@@ -0,0 +1,22 @@
+---
+title: File Drop Zone
+data:
+  description: FileDropZone provides a target area for users to drag and drop files, such as documents or images, and automatically uploads them to the web application.
```

Comment:
they are not technically part of the component, the banner will be documented in the pattern, and the button is part of it but it can be swapped for another trigger component